package.json

```diff
@@ -9,8 +9,8 @@
     "postinstall": "download_deps --package package.json",
     "docs": "documentation build --config etc/documentation.yml --github true --output docs --format html src/**",
     "serve-docs": "documentation serve --config etc/documentation.yml --github true --output docs --format html src/**",
-    "test": "mocha",
-    "test-coverage": "istanbul cover _mocha --report lcovonly -- -R spec",
+    "test": "mocha --recursive",
+    "test-coverage": "istanbul cover node_modules/mocha/bin/_mocha --report lcovonly -- -R spec --recursive test",
```

Comment:
We shouldn't need to specify the whole path to the `_mocha` command with `node_modules/mocha/bin/_mocha` as npm knows where to find it, isn't it working without the full path anymore?